R/error-checks.R

```diff
@@ -75,6 +75,11 @@ validateIsString <- function(object, nullAllowed = FALSE) {
 }
 
 validateIsNumeric <- function(object, nullAllowed = FALSE) {
+  # Only NA values. It is numeric
+  if(all(is.na(object))){

```

Comment:
@pchelle @PavelBal Interestingly, this was not working with NA
NA for some reason is seems as a boolean ??